src/main/java/webserver/RequestHandler.java

```diff
@@ -24,28 +23,37 @@ public void run() {
                 connection.getPort());
 
         try (InputStream in = connection.getInputStream(); OutputStream out = connection.getOutputStream()) {
-
             BufferedReader br = new BufferedReader(new InputStreamReader(in, "UTF-8"));
 
-            String line = br.readLine();
-            StartLine startLine = HttpRequest.getStartLine(line);
+            HttpRequest httpRequest = RequestSeparater.askHttpRequest(br);
+            HttpResponse httpResponse = new HttpResponse();
 
-            if (startLine.getMethod().equals("GET") && startLine.getParamMap() != null) {         // 만약 GET 메서드 요청이 파라미터를 갖고 있다면, User 클래스 생성
-                User user = new User(startLine.getParamMap());
-                logger.debug("{}", user);
-            }
+            String viewName = DispatcherServlet.service(httpRequest, httpResponse);             // 컨트롤러가 반환한 viewName
```

Comment:
1.
```suggestion
            HttpResponse httpResponse = DispatcherServlet.service(httpRequest);             // 컨트롤러가 반환한 viewName
```
요렇게 하교 viewName 도 response 안에 들어있으면 어떨까 생각해 봤습니다.